docs/guides.md

```diff
@@ -1042,23 +1042,29 @@ secureboot-certs install
 Advanced use, not needed by most users.
 :::
 
-`secureboot-certs` also supports installing your own custom certificates. The certs must be in the format of ".auth" files, which may be accomplished using `/opt/xensource/libexec/create-auth`.
+`secureboot-certs` also supports installing your own custom certificates. The certs may be in the following formats:
+
+* DER-encoded certificate
+* PEM-encdoed certificate
+* An auth file (can be created with `/opt/xensource/libexec/create-auth`).
 
 For example, to install a custom PK you may do the following:
 
 ```
 # Create the key and cert using OpenSSL
-openssl req -newkey rsa:4096 -nodes -new -x509 -sha256 -days 3650 -subj "/CN=My PK/" -keyout pk.key -out pk.cer
+openssl req -newkey rsa:4096 -nodes -new -x509 -sha256 -days 3650 -subj "/CN=My PK/" -keyout pk.key -out PK.cer
 
 # Create the .auth file from the DER certificate
 /opt/xensource/libexec/create-auth -k pk.key -c pk.cer PK PK.auth pk.cer
 
 # Enroll it, along with the default certificates, with secureboot-certs
-secureboot-certs install PK.auth default default latest
+secureboot-certs install PK.cer default default latest
 ```
 
 The same procedure may be used to install custom KEK, db, or dbx certs.
 
+To use multiple certificates in one variable (that is, have multiple certificates in PK, KEK, db, or dbx), the certs must be packaged together into a .auth file, see [Use two or more certificates for a Secure Boot variable](#use-two-or-more-certificates-for-a-secure-boot-variable).
```

Comment:
I thought there couldn't be several certificates in PK, according to https://blog.hansenpartnership.com/the-meaning-of-all-the-uefi-keys/